cob_map_accessibility_analysis/ros/src/map_accessibility_analysis_client.cpp

```diff
@@ -33,9 +33,9 @@ int main(int argc, char** argv)
 
   ros::NodeHandle n;
 
-  std::string points_service_name = "/map_accessibility_analysis/map_points_accessibility_check";
-  std::string perimeter_service_name = "/map_accessibility_analysis/map_perimeter_accessibility_check";
-  std::string polygon_service_name = "/map_accessibility_analysis/map_polygon_accessibility_check";
+  std::string points_service_name = "/map_accessibility_analysis/map_accessibility_analysis/map_points_accessibility_check";
+  std::string perimeter_service_name = "/map_accessibility_analysis/map_accessibility_analysis/map_perimeter_accessibility_check";
+  std::string polygon_service_name = "/map_accessibility_analysis/map_accessibility_analysis/map_polygon_accessibility_check";
```

Comment:
you could simply remove the `ns="map_accessibility_analysis"` in the launch file to achieve backwards-compatibility wrt the resulting namespace...

Reply:
I thought it was with double namespace before. But that's fine, I will change that.

Reply:
advertising the topic under the - now private - nodehandle changed the topic/service name of the server...see also that you had to add the extra namespace to the client...!